Atomic/mount.py

```diff
@@ -36,6 +36,11 @@
 
 # Module for mounting and unmounting containerized applications.
 
+DMSETUP_PATH='/usr/sbin/dmsetup'
+LSBLK_PATH='/usr/bin/lsblk'
+MOUNT_PATH='/usr/bin/mount'
+FINDMNT_PATH='/usr/bin/findmnt'
+
```

Comment:
They are in the same location in all of these cases. There could be a problem if other distros picked up the atomic command like debian, maybe.  But for now I think we should fully path these.